digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/GeometryTransform.scala

```diff
@@ -326,38 +312,27 @@ class VKMGeometryTransform {
     val roadPart = validateAndConvertToInt(VkmRoadPart, data)
     val track = validateAndConvertToInt(VkmTrackCode, data)
     val mValue = validateAndConvertToInt(VkmDistance, data)
-    val deviation = convertToDouble(data.get(VkmOffsetToGivenCoordinates))
+    val deviation = Option(0.0)
     if (Track.apply(track).eq(Track.Unknown)) {
       throw new RoadAddressException("Invalid value for Track (%s): %d".format(VkmTrackCode, track))
     }
     RoadAddress(municipalityCode.map(_.toString), road, roadPart, Track.apply(track), mValue, deviation)
   }
 
-  private def mapCoordinates(data: Map[String, Any]) = {
-    val returnCode = data.getOrElse(VkmReturnCode, VkmReturnCodeOk)
-    val error = data.get(VkmError)
-    if (returnCode.toString.toInt == VkmReturnCodeError || error.nonEmpty)
-      throw new RoadAddressException("VKM error: %s".format(data.getOrElse(VkmReturnErrorMessage, error.getOrElse(""))))
```

Comment:
Why were these checks removed?

Reply:
@oskarmalmstrom  because the return json is completed different and the current json don't have that keys.